### PR TITLE
[Core] Minor syntax update

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: src/Core/src/Response.php [hunk 1]
```diff
@@ -373,12 +373,8 @@ private function getResolveStatus(): bool
         }
 
         if ($this->resolveResult instanceof Exception) {
-            if (404 === $code) {
-                $this->logger->info($message ?? $this->resolveResult->getMessage(), $context);
-            } else {
-                $this->logger->error($message ?? $this->resolveResult->getMessage(), $context);
-            }
-
+            $level = 404 === $code ? 'info' : 'error';
```
Comment (bendavies) [inline src/Core/src/Response.php:376]: inline this var?

File: src/Core/src/Response.php [hunk 2]
```diff
@@ -373,12 +373,8 @@ private function getResolveStatus(): bool
         }
 
         if ($this->resolveResult instanceof Exception) {
-            if (404 === $code) {
-                $this->logger->info($message ?? $this->resolveResult->getMessage(), $context);
-            } else {
-                $this->logger->error($message ?? $this->resolveResult->getMessage(), $context);
-            }
-
```
Comment (jderusse) [inline src/Core/src/Response.php:381]: use constants?

Comment (Nyholm) [reply src/Core/src/Response.php:381]: Sure

File: src/Core/src/Response.php (hunk 1 above)
Comment (Nyholm) [reply src/Core/src/Response.php]: Okey.

**Pull request merged**